gemfiles/rails_51.gemfile

```diff
@@ -2,10 +2,10 @@ source "https://rubygems.org"
 
 eval_gemfile(File.expand_path(File.join("..", "Gemfile"), __dir__))
 
-gem "rails", "~> 5.1.x"
+gem "rails", "~> 5.1"
 gem "jquery-ui-rails", "~> 5.0"
-gem "devise", "> 4.x"
-gem "draper", "> 3.x"
+gem "devise", git: "https://github.com/plataformatec/devise"
```

Comment:
4.3 is coming real soon, really https://github.com/plataformatec/devise/issues/4518#issuecomment-300962107